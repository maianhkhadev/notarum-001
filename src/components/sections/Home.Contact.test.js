import React from 'react';
import renderer from 'react-test-renderer';
import Section from './Home.Contact';

it('renders correctly', () => {
  const tree = renderer.create(<Section />).toJSON();
  expect(tree).toMatchSnapshot();
});
