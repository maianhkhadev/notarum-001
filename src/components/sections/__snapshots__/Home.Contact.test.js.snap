// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="section-contact"
>
  <div
    className="section-header"
  >
    <h3
      className="section-title"
    >
      Contact Us
    </h3>
  </div>
  <div
    className="section-content"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-xl-6"
        >
          <h4>
            We'd love to hear from you.
          </h4>
          <p>
            Let us know if you are interested to participate in our beta program. We will then reach out to understand more about your business and requirements.
          </p>
        </div>
        <form
          className="col-xl-6"
        >
          <div
            className="form-group"
          >
            <label
              htmlFor="exampleInputEmail1"
            >
              Email address
            </label>
            <input
              aria-describedby="emailHelp"
              className="form-control"
              id="exampleInputEmail1"
              placeholder="Enter email"
              type="email"
            />
            <small
              className="form-text text-muted"
              id="emailHelp"
            >
              We'll never share your email with anyone else.
            </small>
          </div>
          <div
            className="form-group"
          >
            <label
              htmlFor="exampleInputPassword1"
            >
              Password
            </label>
            <input
              autoComplete="true"
              className="form-control"
              id="exampleInputPassword1"
              placeholder="Password"
              type="password"
            />
          </div>
          <div
            className="form-group form-check"
          >
            <input
              className="form-check-input"
              id="exampleCheck1"
              type="checkbox"
            />
            <label
              className="form-check-label"
              htmlFor="exampleCheck1"
            >
              Check me out
            </label>
          </div>
          <button
            className="btn btn-primary"
            type="submit"
          >
            Submit
          </button>
        </form>
      </div>
    </div>
  </div>
</section>
`;
